src/homepageCounter.js

```diff
@@ -0,0 +1,2 @@
+const homepageCounter = (items) => items.length;
+export default homepageCounter;
```

Comment:
- [ ]  As we are not going to use this file to add more functions, you can move the function to the index.js file